src/components/Dashboard/TodoList.vue

```diff
@@ -701,16 +701,17 @@ export default {
     },
 
     /** Loads a NEW Annual Report todo. */
-    loadAnnualReportTodo (task) {
+    loadAnnualReportTodo (task: any) {
       const todo: FilingIF = task.task.todo
       if (todo?.header && todo?.business) {
         const ARFilingYear: number = todo.header.ARFilingYear
         const subtitle: string = task.enabled && !this.isBComp ? '(including Address and/or Director Change)' : null
 
-        this.taskItems.push({
+        const item: TodoItemIF = {
```

Comment:
I did it this was to type the object. (It identified some issues, which I fixed.)